src/ApiCallJS.js

```diff
@@ -721,19 +732,17 @@ export async function deleteProfile(userId) {
   });
 }
 
-export async function getOrders(user_id) {
+export async function getOrders() {
```

Comment:
Few other methods can have the user_id parameter removed. See ticket comments, I've mentioned a few there e.g. deleteProfile